sklearn/covariance/_shrunk_covariance.py

```diff
@@ -106,12 +106,16 @@ def shrunk_covariance(emp_cov, shrinkage=0.1):
 
     where `mu = trace(cov) / n_features`.
     """
-    emp_cov = check_array(emp_cov)
-    n_features = emp_cov.shape[0]
+    emp_cov = check_array(emp_cov, allow_nd=True)
```

Comment:
I would like to isolate this change in another PR. This is not a simple parameter validation and it requires an entry in the changelog.

We should also isolate the test in a specific test function.